xxhash.h

```diff
@@ -3178,6 +3184,12 @@ enum XXH_VECTOR_TYPE /* fake enum */ {
 #  endif
 #endif
 
+/* __ARM_FEATURE_SVE is only supported by GCC & Clang. */
+#if (XXH_VECTOR == XXH_SVE) && !defined(__ARM_FEATURE_SVE)
```

Comment:
Note : maybe minor, but surely worth mentioning.

This transparent fallback method
will make compilation succeed when `SVE` is selected, but actually not present on compilation target.

But it also means that a developer may think that the library is compiled with `SVE` enabled, because it's instructed to, but actually it's not and uses `SCALAR` instead, and the developer would be none the wiser.

I'm wondering if it would be preferable to let compilation fail instead.

Intermediate position, maybe a warning (though unfortunately, I believe `#warning` is not part of `C90` syntax, and `#pragma warning` depends on compiler support. But then, so does `__ARM_FEATURE_SVE`).

Reply:
OK. I'll update it.